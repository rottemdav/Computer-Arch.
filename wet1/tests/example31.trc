32 2 5 2 local_history local_tables using_share_mid
0xdf703248 T 0x12ee97c
0x544 T 0xee7400
0x4b20da0 T 0x0ac0e4
0x121a3c T 0x5d9b2340
0xdec9a8ec N 0x5474
0xf58 N 0xa1b3e4
0x121a3c N 0x5d9b2340
0x6a0eedd0 N 0xe40
0xdf00374 T 0x294
0xf58 N 0xa1b3e4
0xdf703248 N 0x12ee97c
0xb2a0 N 0xd5069ed4
0x8630 N 0xd26c
0x8630 N 0xd26c
0xdec9a8ec T 0x5474
0x14ec774 N 0x09173240
0x4e028 N 0x35c
0x121a3c N 0x5d9b2340
0xdb6c8 N 0xffc
0x121a3c N 0x5d9b2340
0xf88 N 0x5561c
0x4e028 N 0x35c
0x4b20da0 T 0x0ac0e4
0x4b20da0 T 0x0ac0e4
0xdec9a8ec T 0x5474
0xdb6c8 N 0xffc
0x8e2c7c N 0x81be7bc
0x4b20da0 T 0x0ac0e4
0x6b4 N 0x659a6bc
0x544 N 0xee7400
0x4e028 T 0x35c
0x8630 N 0xd26c
0x4b20da0 N 0x0ac0e4
0xb2a0 T 0xd5069ed4
0x121a3c T 0x5d9b2340
0x6b4 T 0x659a6bc
0x4e028 N 0x35c
0x8630 N 0xd26c
0x8630 N 0xd26c
0xf58 T 0xa1b3e4
0xf58 N 0xa1b3e4
0x121a3c T 0x5d9b2340
0xe780 T 0x998
0x14ec774 T 0x09173240
0xdf00374 T 0x294
0x144 T 0x0c76c
0xdb6c8 T 0xffc
0xdb6c8 T 0xffc
0x144 N 0x0c76c
0xf88 N 0x5561c
0x8630 N 0xd26c
0xf88 T 0x5561c
0x8630 N 0xd26c
0xe780 N 0x998
0x144 N 0x0c76c
0xdf00374 T 0x294
0x8e2c7c T 0x81be7bc
0x6b4 N 0x659a6bc
0xdf703248 T 0x12ee97c
0x4e028 T 0x35c
0x4b20da0 T 0x0ac0e4
0xdec9a8ec T 0x5474
0x121a3c T 0x5d9b2340
0x8e2c7c T 0x81be7bc
0xb2a0 N 0xd5069ed4
0x4b20da0 N 0x0ac0e4
0x6b4 T 0x659a6bc
0xdb6c8 T 0xffc
0x14ec774 T 0x09173240
0xf58 T 0xa1b3e4
0xb2a0 T 0xd5069ed4
0x8630 T 0xd26c
0xe780 T 0x998
0x144 T 0x0c76c
0xdf00374 T 0x294
0xdf703248 T 0x12ee97c
0xf88 T 0x5561c
0x8e2c7c T 0x81be7bc
0x6a0eedd0 T 0xe40
0xdf00374 N 0x294
0xb2a0 T 0xd5069ed4
0xdf703248 N 0x12ee97c
0x544 T 0xee7400
0xdf703248 T 0x12ee97c
0x4b20da0 T 0x0ac0e4
0x121a3c T 0x5d9b2340
0x8630 T 0xd26c
0xdf00374 T 0x294
0xdb6c8 N 0xffc
0x8630 T 0xd26c
0x4b20da0 T 0x0ac0e4
0x6b4 T 0x659a6bc
0x121a3c N 0x5d9b2340
0x8e2c7c N 0x81be7bc
0x6a0eedd0 N 0xe40
0x8630 N 0xd26c
0xf58 N 0xa1b3e4
0x14ec774 T 0x09173240
0x8e2c7c N 0x81be7bc
0xdf00374 N 0x294
0xdf00374 N 0x294
0x4e028 T 0x35c
0x8630 N 0xd26c
0x14ec774 N 0x09173240
0x14ec774 N 0x09173240
0x121a3c N 0x5d9b2340
0x8e2c7c N 0x81be7bc
0xdf00374 T 0x294
0x544 T 0xee7400
0x8e2c7c N 0x81be7bc
0xdb6c8 N 0xffc
0x544 N 0xee7400
0xe780 T 0x998
0x4e028 T 0x35c
0x121a3c N 0x5d9b2340
0x8630 N 0xd26c
0x6b4 N 0x659a6bc
0xdb6c8 N 0xffc
0xdec9a8ec T 0x5474
0xdf00374 T 0x294
0xf58 T 0xa1b3e4
0xdf00374 N 0x294
0x121a3c N 0x5d9b2340
0xdf703248 N 0x12ee97c
0x14ec774 N 0x09173240
0xdec9a8ec T 0x5474
0xdf00374 N 0x294
0xf58 N 0xa1b3e4
0xdec9a8ec N 0x5474
0xdf00374 N 0x294
0x6a0eedd0 T 0xe40
0x144 T 0x0c76c
0xb2a0 T 0xd5069ed4
0xdec9a8ec N 0x5474
0x6b4 N 0x659a6bc
0xf88 T 0x5561c
0x544 N 0xee7400
0xdb6c8 T 0xffc
0xe780 T 0x998
0xf58 T 0xa1b3e4
0xe780 N 0x998
0x6a0eedd0 N 0xe40
0xdf703248 T 0x12ee97c
0xf58 T 0xa1b3e4
0x544 T 0xee7400
0xdf703248 N 0x12ee97c
0xdb6c8 N 0xffc
0x14ec774 T 0x09173240