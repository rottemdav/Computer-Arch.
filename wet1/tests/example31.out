0xdf703248 N 0xdf70324c
0x544 N 0x548
0x4b20da0 N 0x4b20da4
0x121a3c N 0x121a40
0xdec9a8ec N 0xdec9a8f0
0xf58 N 0xf5c
0x121a3c T 0x5d9b2340
0x6a0eedd0 N 0x6a0eedd4
0xdf00374 N 0xdf00378
0xf58 N 0xf5c
0xdf703248 T 0x12ee97c
0xb2a0 N 0xb2a4
0x8630 N 0x8634
0x8630 N 0x8634
0xdec9a8ec N 0xdec9a8f0
0x14ec774 N 0x14ec778
0x4e028 N 0x4e02c
0x121a3c T 0x5d9b2340
0xdb6c8 N 0xdb6cc
0x121a3c T 0x5d9b2340
0xf88 N 0xf8c
0x4e028 N 0x4e02c
0x4b20da0 N 0x4b20da4
0x4b20da0 T 0xac0e4
0xdec9a8ec T 0x5474
0xdb6c8 N 0xdb6cc
0x8e2c7c N 0x8e2c80
0x4b20da0 T 0xac0e4
0x6b4 N 0x6b8
0x544 T 0xee7400
0x4e028 N 0x4e02c
0x8630 N 0x8634
0x4b20da0 T 0xac0e4
0xb2a0 N 0xb2a4
0x121a3c T 0x5d9b2340
0x6b4 N 0x6b8
0x4e028 T 0x35c
0x8630 N 0x8634
0x8630 N 0x8634
0xf58 N 0xf5c
0xf58 T 0xa1b3e4
0x121a3c N 0x121a40
0xe780 N 0xe784
0x14ec774 N 0x14ec778
0xdf00374 N 0xdf00378
0x144 N 0x148
0xdb6c8 N 0xdb6cc
0xdb6c8 T 0xffc
0x144 T 0xc76c
0xf88 N 0xf8c
0x8630 N 0x8634
0xf88 N 0xf8c
0x8630 N 0x8634
0xe780 T 0x998
0x144 T 0xc76c
0xdf00374 T 0x294
0x8e2c7c N 0x8e2c80
0x6b4 T 0x659a6bc
0xdf703248 N 0xdf70324c
0x4e028 T 0x35c
0x4b20da0 N 0x4b20da4
0xdec9a8ec T 0x5474
0x121a3c T 0x5d9b2340
0x8e2c7c T 0x81be7bc
0xb2a0 N 0xb2a4
0x4b20da0 N 0x4b20da4
0x6b4 T 0x659a6bc
0xdb6c8 N 0xdb6cc
0x14ec774 N 0x14ec778
0xf58 T 0xa1b3e4
0xb2a0 N 0xb2a4
0x8630 N 0x8634
0xe780 T 0x998
0x144 T 0xc76c
0xdf00374 N 0xdf00378
0xdf703248 N 0xdf70324c
0xf88 T 0x5561c
0x8e2c7c T 0x81be7bc
0x6a0eedd0 N 0x6a0eedd4
0xdf00374 T 0x294
0xb2a0 T 0xd5069ed4
0xdf703248 T 0x12ee97c
0x544 N 0x548
0xdf703248 T 0x12ee97c
0x4b20da0 N 0x4b20da4
0x121a3c T 0x5d9b2340
0x8630 T 0xd26c
0xdf00374 T 0x294
0xdb6c8 N 0xdb6cc
0x8630 T 0xd26c
0x4b20da0 T 0xac0e4
0x6b4 N 0x6b8
0x121a3c T 0x5d9b2340
0x8e2c7c T 0x81be7bc
0x6a0eedd0 T 0xe40
0x8630 T 0xd26c
0xf58 N 0xf5c
0x14ec774 N 0x14ec778
0x8e2c7c T 0x81be7bc
0xdf00374 N 0xdf00378
0xdf00374 N 0xdf00378
0x4e028 N 0x4e02c
0x8630 T 0xd26c
0x14ec774 N 0x14ec778
0x14ec774 N 0x14ec778
0x121a3c N 0x121a40
0x8e2c7c T 0x81be7bc
0xdf00374 N 0xdf00378
0x544 T 0xee7400
0x8e2c7c N 0x8e2c80
0xdb6c8 N 0xdb6cc
0x544 T 0xee7400
0xe780 N 0xe784
0x4e028 T 0x35c
0x121a3c T 0x5d9b2340
0x8630 N 0x8634
0x6b4 T 0x659a6bc
0xdb6c8 N 0xdb6cc
0xdec9a8ec T 0x5474
0xdf00374 T 0x294
0xf58 T 0xa1b3e4
0xdf00374 T 0x294
0x121a3c T 0x5d9b2340
0xdf703248 N 0xdf70324c
0x14ec774 N 0x14ec778
0xdec9a8ec T 0x5474
0xdf00374 N 0xdf00378
0xf58 N 0xf5c
0xdec9a8ec T 0x5474
0xdf00374 N 0xdf00378
0x6a0eedd0 T 0xe40
0x144 N 0x148
0xb2a0 N 0xb2a4
0xdec9a8ec T 0x5474
0x6b4 T 0x659a6bc
0xf88 T 0x5561c
0x544 N 0x548
0xdb6c8 N 0xdb6cc
0xe780 T 0x998
0xf58 T 0xa1b3e4
0xe780 T 0x998
0x6a0eedd0 N 0x6a0eedd4
0xdf703248 N 0xdf70324c
0xf58 N 0xf5c
0x544 N 0x548
0xdf703248 T 0x12ee97c
0xdb6c8 N 0xdb6cc
0x14ec774 N 0x14ec778
flush_num: 73, br_num: 148, size: 1472b
